### ISSUE TITLE
Contrastive Lossクラス

### ISSUE THREAD

**Issue by Kodai-Uozumi** [bisbislab/bisbislib#2]
## 実装パス
bisbislab.keras.losses.Contrastive Loss
## クラス名
Contrastive_Loss
## 入出力
親クラス：keras.losses.Loss
戻り値：tfa.losses.npairs_loss(tf.squeeze(labels), logits)
